pkg/monitor/monitor/util.go

```diff
@@ -215,9 +215,11 @@ func getMonitorDeploymentSkeleton(sa *core.ServiceAccount, monitor *v1alpha1.Tid
 	monitorLabel := label.New().Instance(monitor.Name).Monitor().Labels()
 	replicas := int32(1)
 
+	labels := label.NewMonitor().Instance(monitor.Name).Monitor().Labels()
```

Comment:
No need to call `Labels()` in the end.
BTW, what does L215 for?

Reply:
`monitorLabel` is for the label of the `tidbmonitor` deployment and `labels` here is for the label selector of `tidbmonitor` deployment